recipe/meta.yaml

```diff
@@ -1,14 +1,13 @@
-{% set version = "0.10.0" %}
-{% set commit = "07f142df646c9070fc9c627004dfdc96914fe487" %}
-{% set checksum = "ec8494f1906a2919d1eaa9445c97f88eacd38695bfbc5ae5f44315bcbbd6e609" %}
+{% set version = "0.11.0" %}
+{% set checksum = "0ac629a7775d86108e403eb66d9f1a3d3bdd6b3a497a86228aa4e8143364b7cc" %}
 
 package:
   name: arrow-cpp
   version: {{ version }}
 
 source:
-  fn: {{ commit }}.tar.gz
-  url: https://github.com/apache/arrow/archive/{{ commit }}.tar.gz
+  fn: apache-arrow-{{ version }}.tar.gz
+  url: https://github.com/apache/arrow/archive/apache-arrow-{{ version }}.tar.gz
```

Comment:
I suppose it'd be better to use the official apache archive instead.

Reply:
Indeed. I think it's OK to use the dist link here since this job doesn't run often